engine/src/main/scala/cromwell/engine/workflow/workflowstore/SqlWorkflowStore.scala

```diff
@@ -13,9 +15,13 @@ import scala.concurrent.{ExecutionContext, Future}
 
 case class SqlWorkflowStore(sqlDatabase: WorkflowStoreSqlDatabase) extends WorkflowStore {
   override def initialize(implicit ec: ExecutionContext): Future[Unit] = {
-    sqlDatabase.updateWorkflowState(
-      WorkflowStoreState.Running.toString,
-      WorkflowStoreState.Restartable.toString)
+          if ( ConfigFactory.load().as[Option[Boolean]]("system.workflow-restart").getOrElse(true)) {
+            sqlDatabase.updateWorkflowState(
+                WorkflowStoreState.Running.toString,
+                WorkflowStoreState.Restartable.toString)
+          } else {
+            Future.successful(())
```

Comment:
it's in the migration.md, as well as in the comments of the conf.reference.  At this point, I think it's a small enough option to leave it there.  However, if/when this becomes a key parameter in standing up several cromwells (e.g. 'reader cromwells' then we should document the suite of parameters required to do that)